code/modules/research/designs/medical_designs.dm

```diff
@@ -811,6 +811,18 @@
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_MEDICAL
 
+/datum/design/cybernetic_heart/anomalock
+	name = "Experimental Cybernetic Heart"
```

Comment:
```suggestion
	name = "experimental cybernetic heart"
```
no caps

far too lame a name for this imo. needs something snazzier, experimental is mid

```suggestion
	name = "voltaic combat cyberheart"
```